iml-agent/src/action_plugins/action_plugin.rs

```diff
@@ -62,7 +62,8 @@ pub fn create_registry() -> action_plugins::Actions {
             action_configure::update_and_write_new_config,
         )
         .add_plugin("is_ntp_configured", is_ntp_configured::is_ntp_configured)
-        .add_plugin("create_ltuer_conf", ltuer::create_ltuer_conf);
+        .add_plugin("create_ltuer_conf", ltuer::create_ltuer_conf)
+        .add_plugin("create_ldev_conf", snapshot::create_ldev_conf);
```

Comment:
Maybe "snapshot" is not the best name? :)

Reply:
I thought about creating a name that had to do with `ldev` but I thought `snapshot` made more sense. The snapshots folder you are referring to is created by `insta` I believe and has unit test snapshots in it. My reasoning behind naming the file `snapshot.rs` is because the actions that live in it are related to snapshots. With that said, I'm open to changing it though.

Reply:
There will be more actions related to lustre snapshots.

I think something like `ldev_conf::create()` would be fine.

Reply:
sure, i'll update.